### ISSUE TITLE
Desire for Helm chart?

### ISSUE THREAD

**Issue by aleveille** [drakkan/sftpgo#1523]
### Describe the solution you'd like

Is there a need or desire for an Helm chart? I see there's already a great Docker image. I could provide the Helm chart and github workflows so that SFTPgo could be installed as an Helm chart

### Describe alternatives you've considered

There's a chart out there: https://artifacthub.io/packages/helm/sagikazarmark/sftpgo
However, there's a lot missing for what I would consider to be a production-ready chart

### What are you using SFTPGo for?

Small business (3-person firm with file exchange?)

**Comment by drakkan** [comment]
If you have time/motivation to send a PR we can try, but I cannot ensure it will merged, it depends on the added complexity.

I have tested the mentioned Helm chart myself and I'm aware of several limitations, additionally it is almost always out of date, for example the current version is vulneable to the Terrapin attack, so it is basically useless.

A well maintained Heml chart may be useful, but I fear that such an addition will increase the maintenance burden even more (if it will be merged it will be also maintained), people will open issues and get angry if we close them because they are invalid.

Basically most users/large companies use SFTPGo because it's free, they claim free support and ignore the request to support the project, I'm tired of this, maybe people deserve barely maintained software like the mentioned Helm chart, this is what they pay for :smile:

**Comment by jcjones87** [comment]
I use a modified version of the linked Helm chart to manage SFTPGo currently. Would be happy to help maintain a version of it here.

**Comment by sagikazarmark** [comment]
I moved my helm chart to this repo: https://github.com/sftpgo/helm-chart

Although I still use it for a pet project, more maintainers are definitely welcome.
